config.yaml

```diff
@@ -59,7 +65,7 @@ submodels:
 
     - name: ice
       model: cice5
```

Comment:
The MOM5 executable also needs to be updated. 

I have made a [comment in the issue with the paths](https://github.com/ACCESS-NRI/access-om2-configs/issues/16#issuecomment-2048709742), but specifically:

```
/g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-qji4nlmr6utrribaiyhewe4je6mifguz/bin/fms_ACCESS-OM.x
```